src/groupmod.c

```diff
@@ -261,6 +261,17 @@ static void grp_update (void)
 			if (NULL != grp.gr_mem[0])
 				grp.gr_mem = dup_list (grp.gr_mem);
 		}
+#ifdef	SHADOWGRP
+		if (NULL != osgrp) {
+			if (!aflg) {
+				sgrp.sg_mem = XMALLOC(1, char *);
+				sgrp.sg_mem[0] = NULL;
+			} else {
+				if (NULL != sgrp.sg_mem[0])
+					sgrp.sg_mem = dup_list (sgrp.sg_mem);
```

Comment:
Please remove the whitespace before the `(`.

Reply:
Done. Note: That superfluous whitespace is also present in several other function calls, e.g. the equivalent one in line 262.

Reply:
Yep, that's something I'm working on.  Slowly.  :)